app/javascript/components/CustomerSearch.js

```diff
@@ -92,32 +92,32 @@ class CustomerApplications extends React.Component {
           </thead>
           <tbody className='rebate-results-table-body'>
             {map(this.state.rebateForms, (rebateForm, key) => {
-              const { property, fields, id } = rebateForm
+              const { property, fields, id } = rebateForm;
               return (
-                <tr key={`${key}-${fields.full_name}`} className='rebate_form.completed'>
+                <tr key={`${key}-${fields.full_name}`} className="rebate_form.completed">
```

Comment:
style issue; on line 55 you used the `{'class'}` syntax which is different to this line